Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -126,6 +128,8 @@ namespace Multiplayer
     AZ_CVAR(AZ::TimeMs, bg_captureTransportPeriod, AZ::TimeMs{1000}, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
         "How often in milliseconds to record transport metrics.");
 
+    AZ_CVAR(bool, sv_multithreadedConnectionUpdates, false, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
```

Comment:
Maybe update MultiplayerSample setreg to keep this enabled? Disabled cvars tend to wilt over time.